sys/arm64/arm64/trap.c

```diff
@@ -158,6 +158,10 @@ cpu_fetch_syscall_args(struct thread *td)
 	struct proc *p;
 	syscallarg_t *ap, *dst_ap;
 	struct syscall_args *sa;
+#if __has_feature(capabilities)
+	char * __capability stack_args = NULL;
```

Comment:
Might be nicer to make this a `syscallarg_t *` here? It's true, unlike RISC-V and MIPS where integers aren't a full capability.